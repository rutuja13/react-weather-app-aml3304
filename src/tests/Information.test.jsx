import { configure, shallow } from "enzyme";
import React from 'react';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Information from "../components/Information";


configure({ adapter: new Adapter() });

jest.mock('../service/api', () => {
    return {
        getData: jest.fn()
    };
});

describe('Information Component', () => {
    it('Snapshot Matching', () => {
        const wrapper = shallow(<Information />);
        expect(wrapper.getElements()).toMatchSnapshot();
    });

    it('Data Component', () => {
        const wrapper = shallow(<Information />);
        expect(wrapper.find('Data')).toBeTruthy();
    });

    it('Typography Component', () => {
        const wrapper = shallow(<Information />);
        expect(wrapper.find('Typography')).toBeTruthy();
    });


});

