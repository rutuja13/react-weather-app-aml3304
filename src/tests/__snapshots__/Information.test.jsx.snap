// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information Component Snapshot Matching 1`] = `
Array [
  null,
]
`;
